tests/test_tox_uv_installer.py

```diff
@@ -60,3 +60,31 @@ def test_uv_install_without_pre_custom_install_cmd(tox_project: ToxProjectCreato
     })
     result = project.run("-vv")
     result.assert_success()
+
+
+def test_uv_install_with_resolution_strategy(tox_project: ToxProjectCreator) -> None:
+    project = tox_project({
+        "tox.ini": """
+    [testenv]
+    deps = tomli
+    package = skip
+    uv_resolution = lowest
+    """
+    })
+    result = project.run("-vv")
+    result.assert_success()
```

Comment:
This is an insufficient test, you actually need to assert that the lowest resolution argument was passed on and not just check the outcome was success. The outcome could be success even with the different strategy.

Reply:
Hopefully reoslved by 14fcbfc.